SwiftPM support broken since update to Swift 5 format

When I try to use AloeStackView with SwiftPM, I get this error:

```
https://github.com/airbnb/AloeStackView.git @ master: error: manifest parse error(s):
/var/folders/4s/rkh28cgs55z2s0cxdsphnfzw0000gn/T/TemporaryFile.3pAAby.swift:8:9: error: type 'SupportedPlatform' has no member 'ios'
       .ios(.v9),
```

This is indicating that the casing of `ios` is wrong and should actually be `iOS`.

--- a/Package.swift
+++ b/Package.swift
@@ -5,7 +5,7 @@ import PackageDescription
 let package = Package(
     name: "AloeStackView",
     platforms: [
-       .ios(.v9),
+       .iOS(.v9),
     ],
     products: [
         .library(

